spacemouse-keys/config_sample.h

```diff
@@ -238,3 +239,35 @@
 #endif
 
 #define DEBOUNCE_KEYS_MS 200  // time in ms which is needed to allow a new button press
+
+// -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+// ENCODER WHEEL
+// Needs the encoder library by Paul Stoffregen.
+// There may be an additional encoder wheel to replace one of the velocities
+#define SIMAXIS 0 
+
+// Axis to replace with encoder
+// 0. None -> disable this feature completely
+// 1. transX
+// 2. transY (zoom in "Forward / Backward" Zoom Direction configuration)
+// 3. transZ (simulates zoom in "Up / Down" Zoom Direction configuration)
+// 4. rotX
+// 5. rotY
+// 6. rotZ
+// (Those are the positions in the velocity array +1, as defined in calibration.h)
+
+// Define the encoder pins
+#define ENCODER_CLK 2
+#define ENCODER_DT 3
+// swap those two pins to change direction of encoder
+
+// To calculate a velocity from the encoder position, the output is faded over so many loop() iterations, as defined in #ECHOES
+// Small number = short duration of zooming <-> Big Number = longer duration of zooming
+// Compare this number with the update frequency of the script, reported by debug=7:
+//    If ECHOES is ca. frequency: the zoom is faded for 1 second.
```

Comment:
what does `ca. frequency` mean ?

Reply:
circa = ca. = approximately.

I change it to: "If ECHOES = frequency:" ...

Reply:
thanks for the explanation

Reply:
Implemented in 39b227a.